docs/command/atlas-clusters-create.txt

```diff
@@ -234,3 +240,11 @@ Examples
 
    # Deploy a cluster or a multi-cloud cluster from a JSON configuration file named myfile.json for the project with the ID 5e2211c17a3e5a48f5497de3:
    atlas cluster create --projectId <projectId> --file myfile.json
+   
+   
+.. code-block::
+   :copyable: false
+
+   # Deploy a three-member sharded cluster with independent shard scaling mode named myRS in GCP for the project with the ID 5e2211c17a3e5a48f5497de3:
+   atlas cluster create myRS --projectId 5e2211c17a3e5a48f5497de3 --provider GCP --region EASTERN_US --members 3 --tier M10  --mdbVersion 5.0 --diskSizeGB 10 --autoScalingMode independentShardScaling
```

Comment:
[Not-blocking] do we plan to provide an alias like ISS for the flag? independentShardScaling is quite long and easy to misspell

Reply:
nice point, I'll check w/ Jonny! Did we ever provide alias for flags in the past?